_posts/2015-08-01-Why-you-should-always-follow-your-dreams.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+name: Why-you-should-always-follow-your-dreams!
+title: Why you should always follow your dreams!
+date: 2015-08-01 12:00:00 +00:00
+author: Ana Nogal
+image:
+    src: /assets/img/custom/blog/follow-your-dreams.jpg
+    attribution:
+        text: Follow your dreams
+tags:
+- dreams
+- codurance
+- carrer
+- culture
+---
+I started at [Codurance](http://codurance.com/) a month and a half ago  as a C# developer. I was glad to have the opportunity to work in a place, where people value your effort to be a better developer. They set you some goals that you are expect to achieve on your own, with their support.
+
+I was happy with my position, I had been away from C# for more than 4 years. This was for me a great opportunity to catch up with my main language. But my passion is with iOS. On my own time, I started to apply all the techniques that I was learning in C# to iOS.  You can have a look [here](http://wp.me/p4i8Xl-6y) or [here](http://codurance.com/2015/07/13/How-to-start-doing-TDD-in-iOS/) were I share my progress.
```

Comment:
When linking to other web pages, it's nice to have some context in the link. Perhaps we could change this to:

    I focused especially on [honing my TDD skills](http://www.ananogal.com/blog/starting-your-tdd-journey/) and [applying TDD to iOS](http://codurance.com/2015/07/13/How-to-start-doing-TDD-in-iOS/).